packages/types/src/srml/deprecated/definitions.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+export default {
```

Comment:
We actually need a "type is defined, but not found in metadata" check. These were found by manual inspection, and surely is not all available.